### PR TITLE
Revert "HTML escape items that are passed as JSON to the front end"

### PR DESCRIPTION
**Pull request by benrudolph**
Reverts dimagi/commcare-hq#9793 @TylerSheffels

### CODE REVIEW

File: corehq/apps/hqwebapp/tests/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-from .test_hq_shared_tags import TestCaseTag, TestJsonTag
+from .test_hq_shared_tags import TestCaseTag
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/tests/__init__.py]: F401 'TestCaseTag' imported but unused

**Comment by czue** [comment]
what happened here?

**Comment by benrudolph** [comment]
this is what i was referring to in the slack team chat: https://dimagi.slack.com/archives/web-team/p1452193843000134

it was escaping things that was causing the page to break

**Comment by TylerSheffels** [comment]
@czue We depend on passing html elements through on some pages and this was breaking them

**Comment by czue** [comment]
:+1: thanks for quick responses

**Pull request merged**